.drone.yml

```diff
@@ -1,18 +1,48 @@
+---
 kind: pipeline
-type: docker
+type: kubernetes
 name: default
 
+trigger:
+  branch:
+    - master
+
```

Comment:
A global trigger only for master seems like it would make the development process harder.   e.g. Tests should run in all cases to give the quickest feedback loop.

Reply:
I totally agree, updating it.